packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -379,6 +379,7 @@ export const EVENT_TRIGGERS = {
   MEETING_RECONNECTION_STARTING: 'meeting:reconnectionStarting',
   MEETING_RECONNECTION_SUCCESS: 'meeting:reconnectionSuccess',
   MEETING_RECONNECTION_FAILURE: 'meeting:reconnectionFailure',
+  MEETING_RESHARE_NEEDED: 'meeting:reshareNeeded',
```

Comment:
Should we just trigger share:stopped with reason as reconnect !?

Reply:
That works!